<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>521</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>600</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="homeTab">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>311</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>COMPRESSO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Copperplate Gothic Light</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Your ultimate compression partner </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="compressbtn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>330</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string> Compress</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/icons/minimize-solid.svg</normalon>
       </iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="decompressbtn">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>330</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string> Decompress</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/icons/maximize-solid.svg</normalon>
       </iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CompTab">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Copperplate Gothic Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select   a   file   to   Compress</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fileBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> Select file</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/icons/file-import-solid.svg</normalon>
       </iconset>
      </property>
     </widget>
     <widget class="QProgressBar" name="cProgressbar">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>671</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="CompressFileBtn">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>370</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Compress</string>
      </property>
     </widget>
     <widget class="QPushButton" name="homeBtn_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string> Home</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/icons/house-solid.svg</normalon>
       </iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="filePathbox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>File Path</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DecompTab">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Copperplate Gothic Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select   a   file   to  Decompress</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="filePathbox_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>190</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>File Path</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fileBtn_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> Select file</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/icons/file-import-solid.svg</normalon>
       </iconset>
      </property>
     </widget>
     <widget class="QProgressBar" name="dCompressbar">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>671</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="DecompressFileBtn">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>370</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Decompress</string>
      </property>
     </widget>
     <widget class="QPushButton" name="homeBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string> Home</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/icons/house-solid.svg</normalon>
       </iconset>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
